beacon-chain/db/kv/schema.go

```diff
@@ -46,8 +46,8 @@ var (
 
 	// Below keys are used to identify objects are to be fork compatible.
 	// Objects that are only compatible with specific forks should be prefixed with such keys.
-	altairKey = []byte("altair")
-	mergeKey  = []byte("merge")
+	altairKey    = []byte("altair")
+	bellatrixKey = []byte("bellatrix")
```

Comment:
This is going to break everyone on testnets. Do not rename database keys without a migration.

Reply:
Reverted